code/datums/quirks/_quirk.dm

```diff
@@ -177,26 +177,33 @@
  */
 /mob/living/proc/get_quirk_string(medical, category = CAT_QUIRK_ALL) //helper string. gets a string of all the quirks the mob has
 	var/list/dat = list()
+	
+	// The health analyzer will first check if the target is a changeling, and if they are, load the quirks of the person they're disguising as.
+	var/target_quirks = quirks
+	var/datum/antagonist/changeling/target_changeling = mind.has_antag_datum(/datum/antagonist/changeling)
+	if(target_changeling)
+		target_quirks = target_changeling.current_profile.quirks
```

Comment:
To be honest I hate this solution as well. Should I just make it so it copies all of the target's quirks when you shapeshift into them? Wouldn't have to add anything to _quirk.dm and I could get rid of the mimicable_quirks list as well.

Reply:
I don't have any knowledge on how signalizing works or what it even is, but I'm curious.

Still, I am fine with skipping this part of the PR for now, as it leaves out texts like "Subject is blind!" and brain traumas, so it will need more work either way.